docs/source/release/v5.1.0/ui.rst

```diff
@@ -19,7 +19,6 @@ See :doc:`mantidplot`.
 
 MantidWorkbench
 ---------------
```

Comment:
Are you using `MantidWorkbench` `workbench`, `Workbench`? `MantidPlot` has been a fairly consistently used for comparison, but there are a range of usages of the former in this PR.

Reply:
I agree we should remove refences to MantidWorkbench, as it has never been that it is just a term that has gotten into our release notes.